charts/qiskit-serverless/values.yaml

```diff
@@ -109,8 +109,8 @@ gateway:
 kuberayOperatorEnable: true
 kuberay-operator:
   image:
-    repository: kuberay/operator
-    tag: v1.0.0
+    repository: quay.io/kuberay/operator
```

Comment:
Curious about this change @psschwei , do we need to change the `Chart.yaml` too?

```
- name: kuberay-operator
```

What are we doing with this change exactly? (Sorry, I tried to find it in the documentation but I didn't find any reference).

Reply:
Ray now publishes their container images to quay, not docker hub, so we need to update the image name:
https://github.com/ray-project/kuberay/releases/tag/v1.1.0

Reply:
Awesome, thanks @psschwei . I also was taking a look into this link that they were mentioning in that release:

https://docs.ray.io/en/master/cluster/kubernetes/k8s-ecosystem/kueue.html#kuberay-kueue

Really interesting and probably related with your issue #1337